sweeper_wallet.go

```diff
@@ -0,0 +1,25 @@
+package lnd
+
+import (
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/lightningnetwork/lnd/lnwallet"
+)
+
+// sweeperWallet is a wrapper around the LightningWallet that implements the
+// sweeper's Wallet interface.
+type sweeperWallet struct {
```

Comment:
Could the method `CancelRebroadcast` be added to `*lnwallet.LightningWallet` directly?

Reply:
+1

Reply:
It's sort of already there as it's within the main config/interface. Goal here was to add another layer of indirection we can use ti give the sweeper more access to internal wallet state, which seems to be a requirement to properly fixing the anchor issues.